recipes/rmf/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir build && cd build
+cmake -DCMAKE_BUILD_TYPE=Release \
+      -G Ninja \
+      -DCMAKE_PREFIX_PATH=${PREFIX} \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX} \
```

Comment:
Use `CMAKE_ARGS` instead which also contains some other useful settings for conda-forge builds.
```suggestion
      ${CMAKE_ARGS} \
```